plume/src/main/kotlin/io/github/plume/oss/passes/graph/BaseCPGPass.kt

```diff
@@ -361,6 +365,7 @@ class BaseCPGPass(private val g: BriefUnitGraph) {
             SootToPlumeUtil.createIdentifierVertex(it.value, currentLine, currentCol, unit.useBoxes.indexOf(it)).apply {
                 addToCache(it.value, this)
                 builder.addEdge(callVertex, this, RECEIVER)
+                builder.addEdge(callVertex, this, ARGUMENT)
```

Comment:
This is failing `testing/src/test/scala/..../querying/CallTests::"should allow traversing from call to arguments"` because it duplicates an argument edge. 

Line 132 `projectCallArg` already adds ARGUMENT edges here

Reply:
As Following code:
https://github.com/ShiftLeftSecurity/codepropertygraph/blob/024f1cda27296638ec8d4fd47ecc60b9ae39c542/semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/compat/argumentcompat/ArgumentCompat.scala#L28

All AST Nodes from Call or Return nodes are treated as ARGUMENT.

So I added Argument edge for them.

Reply:
Ah I see - in that case I'm okay with it. The Scala tests just need to be adjusted still and once the PR is green I'll merge